res4.md

```diff
@@ -0,0 +1,40 @@
+| Test Name | Parameter | h2o | pico.v | veb |
```

Comment:
Remove all the files that are Not necessary for the bench.

Are only noisy files.